server/server/src/lib.rs

```diff
@@ -309,8 +303,6 @@ pub async fn start_server(
 
                 // restart the server in next loop
                 info!("Restart server");
-                // use DB settings in next restart
-                prefer_config_settings = false;
```

Comment:
I wanted to disallow being able to sync from different site when we've already initialised. This would avoid data being combined for two different sites. Without this change it's possible for developer to initialise using UI setting and then after that overwrite those settings with config (which would cause data for two sites to be present and would cause potential bugs to be reported). 

So i didn't see why we would ever want to overwrite sync settings already in db with config from file ?
It's still possible with this change to initialise from config from file (as long as there is no sync settings in db), although this is still dangerous as you can change config in file after initialising with different config from file. I'll add an issue before merging this.